azure-pipelines-dependency-graph-testing.yml

```diff
@@ -79,7 +82,7 @@ stages:
 
     jobs:
       - job: lint
-        condition: or(eq(stageDependencies.scope_check.changes.outputs['CheckChanges.GEChanged'], true), eq(variables.isMain, true))
+        condition: eq(stageDependencies.scope_check.changes.outputs['CheckChanges.GEChanged'], true)
```

Comment:
As we're now limited to `develop` and `pre_pr` branches, we no longer to check `main`.